app/code/Magento/CatalogSearch/Controller/Advanced/Result.php

```diff
@@ -32,33 +25,35 @@ class Result extends \Magento\Framework\App\Action\Action
      *
      * @param Context $context
      * @param ModelAdvanced $catalogSearchAdvanced
-     * @param UrlFactory $urlFactory
```

Comment:
Due to Backwards Compatibility Policy we can not remove any arguments from the constructor at the moment